apps/src/code-studio/assets/SpriteManagementDirectory.jsx

```diff
@@ -1,17 +1,68 @@
-import React from 'react';
+import React, {useState} from 'react';
+import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
+import {moveDefaultSpriteMetadataToProduction} from '@cdo/apps/assetManagement/animationLibraryApi';
+import StatusCheckmarkIcon, {
+  iconStatus
+} from '@cdo/apps/code-studio/components/StatusCheckmarkIcon';
 
-export default class SpriteManagementDirectory extends React.Component {
-  render() {
-    return (
-      <div>
-        <h1>Manage Sprite Lab Sprites</h1>
-        <h3>
-          <a href="/sprites/sprite_upload">Upload New Sprites</a>
-        </h3>
-        <h3>
-          <a href="/sprites/default_sprites_editor">Edit Default Sprites</a>
-        </h3>
-      </div>
+export default function SpriteManagementDirectory() {
+  const [moveChangesStatus, setMoveChangesStatus] = useState(
+    iconStatus.noAction
+  );
+
+  const moveChangesToProduction = () => {
+    //Files to move to production folder: defaultSprites.json
+    moveDefaultSpriteMetadataToProduction()
+      .then(() => {
+        setMoveChangesStatus(iconStatus.success);
+      })
+      .catch(err => {
+        setMoveChangesStatus(iconStatus.failure);
+        console.log(err);
+      });
+  };
+
+  const confirmReleaseChangesToLevelbuilder = () => {
+    let shouldRelease = confirm(
+      'This will release all the sprites you have added and updates to the default sprite list to ' +
+        'production. Are you sure?'
     );
-  }
+    if (shouldRelease) {
+      moveChangesToProduction();
+    }
+  };
+
+  return (
+    <div>
+      <h1>Manage Sprite Lab Sprites</h1>
+      <h3>
+        <a href="/sprites/sprite_upload">Upload New Sprites</a>
+      </h3>
+      <h3>
+        <a href="/sprites/default_sprites_editor">Edit Default Sprites</a>
+      </h3>
+      <div style={styles.pageBreak}>
+        <Button
+          text="Release Changes to Production"
+          color={Button.ButtonColor.red}
+          onClick={confirmReleaseChangesToLevelbuilder}
+          style={styles.button}
+        />
+        <StatusCheckmarkIcon displayStatus={moveChangesStatus} />
+      </div>
+    </div>
+  );
 }
+
+const styles = {
+  pageBreak: {
+    borderTop: `1px solid ${color.dark_slate_gray}`,
+    display: 'flex',
+    alignItems: 'flexStart'
```

Comment:
out of curiosity, is 'flex-start' different from 'start' in this context? from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items) they sound the same, but wondering if you noticed a difference when deciding which to use